### DISCUSSION TITLE
Everything is set up and it cant find python?

### DISCUSSION THREAD

**Discussion by tjohnston1** [Zero6992/chatGPT-discord-bot#270] [Q&A]
Cant run the bot because i continue getting the same error upon trying to run.

"Python was not found; run without arguments to install from the Microsoft Store, or disable this shortcut from Settings > Manage App Execution Aliases."

I have python installed and the vscode extension as well.

How do I get this to run if "python3 main.py" or "python main.py" don't work?

**Comment by tjohnston1** [comment]
Now it says that it cant locate the file and there is no such file........wth


can't open file 'C:\\Users\\trejo\\Downloads\\chatGPT-discord-bot-main\\main.py': [Errno 2] No such file or directory
PS C:\Users\trejo\Downloads\chatGPT-discord-bot-main>

**Comment by brezden** [comment]
Have you tried to use `py main.py` in your terminal? If you have and you are using Visual Studio Code, then you can press "Ctrl+shift+p" and enter "Select Interpreter" and make sure you are running the right python version.
